### PR TITLE
MxModelReflection -> Mx Model Reflection

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/appstore/modules/oidc.md [hunk 1]
```diff
@@ -86,19 +86,19 @@ If you are migrating from the community edition of the module ([OpenIDConnect Si
...
 5. Follow the instructions in [OIDC App Configuration](#app-configuration) to set up your app.
 
-### 3.1 Installing MxModelReflection{#mxmodelreflection}
+### 3.1 Installing Mx Model Reflection{#mxmodelreflection}
 
-Once the MxModelReflection has been imported into your app, you need to configure it.
+Once the Mx Model Reflection module has been imported into your app, you need to configure it.
 
-1. Add the page `MxObjects_Overview` from the `MxModelReflection` module to the Navigation menu
+1. In the **App Explore**r, add the page **MxObjects_Overview** from the **MxModelReflection** folder to the Navigation menu.
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/modules/oidc.md:93]: @Luyao-Zhang-1 formatting issue with the bold here and "r"

Comment (Luyao-Zhang-1) [reply content/en/docs/appstore/modules/oidc.md:93]: @Adam-Dupaski Thanks for catching it! I corrected it in the development branch.

**Comment by MarkvanMents** [comment]
Thanks for doing this Luyao.
I thought it was my job, but you had already merged the improvements before I got as far as doing my review.
So, obviously, I didn't see that there were problems.

**Comment by Luyao-Zhang-1** [comment]
I thought they were quick wins, and I had time, so I just went ahead and fixed them. 
But I should have told people in the thread that I was fixing them, to avoid the duplicate review. I will remember to do it next time. : )

**Pull request merged**